Issue 52 patch

LAGO-Support opened the pull request:
This fix will take into account changes in source *.dat files when applying datum conversion. It will now additionally look at the file name to check if a datum conversion is necessary.

LAGO-Support committed:
adjust datum conversion for new files

This change will let the datum conversion account for new file names and inputs where files may have values in one datum and a different header datum.

LAGO-Support committed:
fix a typo

Made a typo

kirenbahm commented:
This code doesn't work for the case where vertical datum is NGVD29 and datatype is stage_navd88.  When these conditions exist, the script incorrectly changes the ground surface elevation value for the station, where it should leave the GSE as it is.
I will email a sample input file to test with for station SD.

LAGO-Support committed:
fix edge case where header changed

In the case that the file name contains "stage_NAVD88" and the header datum is "NGVD29" the header Ground Surface Elevation will remain unchanged as it is already i the correct datum while he time series values will be converted.
